Library/Homebrew/cmd/tap.rb

```diff
@@ -2,16 +2,20 @@ module Homebrew
   def tap
     if ARGV.empty?
       each_tap do |user, repo|
-        puts "#{user.basename}/#{repo.basename.sub("homebrew-", "")}" if (repo/".git").directory?
+        if (repo/".git").directory?
+          puts "#{user.basename}/#{repo.basename.sub("homebrew-", "")}"
+        end
```

Comment:
Nope. I found it easier to read when I wrote my version as a method, but if you like I'll go back to the old. Two secs for a PR.